packages/node/src/indexer/indexer.manager.ts

```diff
@@ -467,36 +454,33 @@ export class IndexerManager {
     handlers: SubqlRuntimeHandler[],
     blockContent: BlockWrapper,
   ): Promise<void> {
-    const block = blockContent.getBlock();
-    // const extrinsics = substrateBlockContent.getExtrinsincs();
-    // const events = substrateBlockContent.getEvents();
     for (const handler of handlers) {
       switch (handler.kind) {
         case SubqlHandlerKind.Block:
           //  if (SubstrateUtil.filterBlock(block, handler.filter)) {
-          await vm.securedExec(handler.handler, [block]);
+          await vm.securedExec(handler.handler, [blockContent]);
           //  }
           break;
-        // case SubqlHandlerKind.Call: {
-        //   const filteredExtrinsics = SubstrateUtil.filterExtrinsics(
-        //     extrinsics,
-        //     handler.filter,
-        //   );
-        //   for (const e of filteredExtrinsics) {
-        //     await vm.securedExec(handler.handler, [e]);
-        //   }
-        //   break;
-        // }
-        // case SubqlHandlerKind.Event: {
-        //   const filteredEvents = SubstrateUtil.filterEvents(
-        //     events,
-        //     handler.filter,
-        //   );
-        //   for (const e of filteredEvents) {
-        //     await vm.securedExec(handler.handler, [e]);
-        //   }
-        //   break;
-        // }
+        case SubqlHandlerKind.Call: {
+          const filteredExtrinsics = SubstrateUtil.filterExtrinsics(
+            (blockContent as SubstrateBlockWrapped).getExtrinsincs(),
+            handler.filter,
+          );
+          for (const e of filteredExtrinsics) {
+            await vm.securedExec(handler.handler, [e]);
+          }
+          break;
+        }
+        case SubqlHandlerKind.Event: {
+          const filteredEvents = SubstrateUtil.filterEvents(
+            (blockContent as SubstrateBlockWrapped).getEvents(),
+            handler.filter,
+          );
+          for (const e of filteredEvents) {
+            await vm.securedExec(handler.handler, [e]);
+          }
+          break;
+        }
         default:
       }
```

Comment:
So all the handlers are Substrate related. 
In case of avalanche, nothing happens ? There is no Avalanche handlers at this point ?